.github/workflows/daily.yaml

```
@@ -43,3 +43,12 @@ jobs:
           rm /tmp/requirements.txt
       - name: Run Daily Integration Test
         run: tox -vv -e daily-integration-test-${{ matrix.task }}
+
+      - name: Create Issue if Tests Fail
+        if: steps.run_tests.outputs.exit_code != '0'
+        uses: peter-evans/create-issue-from-file@v4
```

Please pin to a specific hash, maybe we can use the latest version.
```suggestion
        uses: peter-evans/create-issue-from-file@e8ef132d6df98ed982188e460ebb3b5d4ef3a9cd  # v5.0.1
```